tests/integration/test_failure_output.py

```diff
@@ -101,3 +254,37 @@ def test_failure_output_after_state_change_for_mk8s(self, single_failing_test_fo
         # Test
         assert len(test_runs) == 1
         pytest.helpers.assert_qtest_property(test_runs[0], 'Failure Output', '')
+
+    # noinspection PyUnresolvedReferences
+    def test_truncated_failure_output_for_asc(self, multi_line_failure_message_for_asc):
+        """Verify the qTest 'Failure Output' test log field is populated with the correct message for a single test
+        failing from the "asc" CI environment.
+        """
+
+        # Setup
+        multi_line_failure_message_for_asc.assert_invoke_zigzag()
+        test_runs = multi_line_failure_message_for_asc.tests[0].qtest_test_runs
+
+        # Expectations
+        test_failure_msg_regex_exp = r"E       assert 'Ubuntu 14\.04 LTS' in ''"
```

Comment:
Shouldn't these tests verify that the entire failure message was written to the log rather than just the error line?